pages/design/atomic-template-empty.md

```diff
@@ -41,42 +44,42 @@ _There are no major accessibility support issues known for this rule._
 
 ### Passed
 
-#### Passed example 1
+#### Passed Example 1
 
 Description...
 
 ```html
 <!-- code -->
 ```
 
-#### Passed example 2
+#### Passed Example 2
 
 ...
 
 ### Failed
 
-#### Failed example 1
+#### Failed Example 1
 
 Description...
```

Comment:
Again: Should this be a comment?

Reply:
Don't think so, this is quite obvious in my opinion.